cmd/find.go

```diff
@@ -375,7 +379,9 @@ func watchEvents(ctx *cli.Context, clnt Client, params watchParams, alias string
 
 // fileContentMatch is used to take the params passed to find, in addition to the current
 // file and call the appropriate "pattern matching methods"
-func fileContentMatch(fileContent contentMessage, orOp bool, ctx *cli.Context) bool {
+func fileContentMatch(ctx *cli.Context, fileContent contentMessage) bool {
+	orOp := ctx.Bool("or")
+
 	match := true
 
 	if (match && !orOp) && ctx.String("ignore") != "" {
```

Comment:
Since both of the operations are `&&`, there is no need to have parenthesis around `(match && !orOp)`.